core/drivers/firewall/stm32_rif.c

```diff
@@ -0,0 +1,124 @@
+// SPDX-License-Identifier: BSD-2-Clause
+/*
+ * Copyright (c) 2021-2024, STMicroelectronics
+ */
+
+#include <assert.h>
+#include <drivers/stm32_rif.h>
+#include <drivers/stm32mp_dt_bindings.h>
+#include <kernel/panic.h>
+#include <mm/core_memprot.h>
+
+static uint32_t get_scid_mask(unsigned int nb_cid_supp)
```

Comment:
I got a bit confused with this. scid refers to semaphore CID or static CID.
I think a brief inline description of this local function is welcome.